lms/static/sass/shared/_modal.scss

```diff
@@ -62,6 +62,11 @@
     padding-bottom: ($baseline/2);
     position: relative;
 
+    p {
+      font-size: .9em;
```

Comment:
NIT: this looks like something that was added in this work. Is that the case? If so, why not use a typography extend from https://github.com/edx/edx-platform/blob/master/lms%2Fstatic%2Fsass%2Felements%2F_typography.scss?